Add cindyliu923-kubernetes-challenge.md

cindyliu923 opened the pull request.

cindyliu923 committed:
Add cindyliu923-kubernetes-challenge.md

OliverMensahDev reviewed (changes requested):
Awesome work @cindyliu923 .

Kindly include in your write-up some screenshots of the DO dashboard or screenshots of a few commands that you used in creating the DO Kubernetes cluster.

cindyliu923 commented:
Hi @OliverMensahDev,
I changed my [readme](https://github.com/cindyliu923/kubernetes-challenge-app#readme)
by the way, I removed the cluster 3 hours ago before your review and without any  screenshots 😂

thank you ❤️
